ui/components/multichain/pages/page/index.scss

```diff
@@ -1 +1,40 @@
 @import 'components/';
+
+.multichain-page {
+  $height-screen-sm-max: 100%;
+  $width-screen-sm-min: 85vw;
+  $width-screen-md-min: 80vw;
+  $width-screen-lg-min: 62vw;
+
+  &__inner-container {
+    @include screen-sm-max {
+      height: $height-screen-sm-max;
+    }
+
+    @include screen-sm-min {
+      width: $width-screen-sm-min;
+    }
+
+    @include screen-md-min {
+      width: $width-screen-md-min;
+    }
+
+    @include screen-lg-min {
+      width: $width-screen-lg-min;
+    }
+  }
```

Comment:
Have you consulted @kylefiedler on these values? I think because most of our screens are designed for popup view it's unlikely that we actually want the container to take up 62%/80%/85% of the screen's width. I think we just give a max value here that takes into account the larger typography. Otherwise we will end up with extremely wide button at large screen sizes. 

![Screenshot 2023-10-31 at 2 36 09 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/15f8421a-a878-42d6-b007-7769b9cc73ae)

```suggestion
   max-width: 480px;
```

Reply:
`480px` should work for most of these pages https://www.figma.com/file/IXyyE5ZNzWrNhnIesQbmal/Page%2FGrid-Audit-and-Insight-Report?type=whiteboard&node-id=0%3A1&t=esrCbADLCDq48qJs-1

Reply:
I like long buttons I cannot lie…

I'm good with this, the one page that 480 doesn't work for is the settings page. I think that is responsive though and turns the sidebar nav into it's own page?

Reply:
cool, added in [8a194b3](https://github.com/MetaMask/metamask-extension/pull/21507/commits/8a194b3e874efb041d41c40091a2f52218c5a914)